functests/1_performance/cpu_management.go

```diff
@@ -380,3 +412,44 @@ func getStressPod(nodeName string) *corev1.Pod {
 		},
 	}
 }
+
+func getTestPodWithAnnotations(annotations map[string]string) *corev1.Pod {
+	testpod := pods.GetTestPod()
+	testpod.Annotations = annotations
+	testpod.Namespace = testutils.NamespaceTesting
+
+	cpus := resource.MustParse("1")
+	memory := resource.MustParse("256Mi")
+
+	// change pod resource requirements, to change the pod QoS class to guaranteed
+	testpod.Spec.Containers[0].Resources = corev1.ResourceRequirements{
+		Limits: corev1.ResourceList{
+			corev1.ResourceCPU:    cpus,
+			corev1.ResourceMemory: memory,
+		},
+	}
+
+	runtimeClassName := components.GetComponentName(profile.Name, components.ComponentNamePrefix)
+	testpod.Spec.RuntimeClassName = &runtimeClassName
+	testpod.Spec.NodeSelector = map[string]string{testutils.LabelHostname: workerRTNode.Name}
+
+	return testpod
+}
+
+func deleteTestPod(testpod *corev1.Pod) {
+	// it possible that the pod already was deleted as part of the test, in this case we want to skip teardown
+	key := types.NamespacedName{
+		Name:      testpod.Name,
+		Namespace: testpod.Namespace,
+	}
+	err := testclient.Client.Get(context.TODO(), key, testpod)
+	if errors.IsNotFound(err) {
+		return
+	}
+
+	err = testclient.Client.Delete(context.TODO(), testpod)
+	Expect(err).ToNot(HaveOccurred())
+
+	err = pods.WaitForDeletion(testpod, 60*time.Second)
```

Comment:
It happened to me I had to wait more :(, maybe my cluster is really slow)

Reply:
It is existing logic from other tests, not sure we need to update it (at least for now)

Reply:
ack